src/main/java/uk/co/ramp/io/ContactReader.java

```diff
@@ -45,6 +45,7 @@ public ContactReader(
 
   List<ImmutableContactEvent> resampleContacts(List<ImmutableContactEvent> contactEvents) {
     int steps = properties.timeStepsPerDay();
+    int offset = properties.dayOffset().orElse(0);
```

Comment:
I don't know if you're generally using GitHub's "squash commits and merge" option, but it is probably a good idea here.  It would be good to mention the addition of the "dayOffset" parameter in the new commit message, as well as explaining the capacity and app uptake changes.

Also, I don't know why `input/contacts.csv` was changed in this PR, but that should be explained too.

Reply:
contact.csv didn't have the label field, so I added it.